src/transport/net_ib.cc

```diff
@@ -561,7 +546,7 @@ static bool ncclMlx5dvDmaBufCapable(ibv_context *context){
 
 ncclResult_t ncclIbMakeVDeviceInternal(int* d, ncclNetVDeviceProps_t* props) {
   if (ncclParamIbMergeNics() == 0 && props->ndevs > 1) {
-    INFO(NCCL_NET, "NET/IB : Skipping makeVDevice, NCCL_IB_MERGE_NICS=0");
+    WARN("NET/IB : Skipping makeVDevice, Please set NCCL_IB_MERGE_NICS=1");
```

Comment:
```suggestion
    INFO(NCCL_NET, "NET/IB : Skipping makeVDevice, NCCL_IB_MERGE_NICS=0");
```
This isn't a warning case, the user may rightfully choose to set NCCL_IB_MERGE_NICS to 0. Let's leave the INFO log as-is.

Reply:
When user is choosing NCCL_IB_MERGE_NICS = 0 and NCCL_NET_FORCE_MERGE="a,b;c,d;e,f .."
RCCL is failing due to conflicting inputs from user. It was difficult to trace the root cause without this warning. One of these environment variables should be of higher priority and In case of conflict, a warning or silently succeeding should be implemented.